### ISSUE TITLE
Release 0.7.8

### ISSUE THREAD

**Issue by hodgestar** [edgewall/genshi#75]
Todo items for 0.7.8:

- [X] Check that master works nicely.
- [X] Remove pytest regex escape warnings.
- [X] Update the changelog.
- [X] Update the 0.7.x branch and bump the version.
- [X] Create a GitHub release.
- [x] Run the release action.

**Comment by hodgestar** [comment]
@jaraco If you're happy to check that the master branch works nicely, I can trigger the release GitHub action.

**Comment by jaraco** [comment]
Looks great. I ran my website using the latest version and it runs without warnings and renders nicely:

```
 @ pip-run git+https://github.com/edgewall/genshi . -- -m cherrypy -i jaraco.site.run
Unable to import openid modules - OpenID support disabled
[13/Jun/2024:17:56:27] ENGINE Listening for SIGTERM.
[13/Jun/2024:17:56:27] ENGINE Listening for SIGHUP.
[13/Jun/2024:17:56:27] ENGINE Listening for SIGUSR1.
[13/Jun/2024:17:56:27] ENGINE Bus STARTING
[13/Jun/2024:17:56:27] ENGINE Started monitor thread 'Autoreloader'.
[13/Jun/2024:17:56:27] ENGINE Serving on http://:::8080
[13/Jun/2024:17:56:27] ENGINE Bus STARTED
::ffff:127.0.0.1 - - [13/Jun/2024:17:56:34] "GET / HTTP/1.1" 200 12214 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:56:34] "GET /static/css/style.css HTTP/1.1" 200 2796 "http://localhost:8080/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:57:37] "GET /projects/ HTTP/1.1" 200 23158 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:57:42] "GET /charts HTTP/1.1" 404 1546 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:58:09] "GET /charts/plot HTTP/1.1" 200 29414 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:58:12] "GET / HTTP/1.1" 200 12214 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
::ffff:127.0.0.1 - - [13/Jun/2024:17:58:12] "GET /static/css/style.css HTTP/1.1" 200 2796 "http://127.0.0.1:8080/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0"
```

**Comment by hodgestar** [comment]
Thanks @jaraco.

I've released 0.7.8. Would you mind giving it a try?

**Comment by jaraco** [comment]
Everything's working great!
